core-web/libs/dotcms-scss/angular/_forms.scss

```diff
@@ -60,6 +60,7 @@
     display: block;
     font-size: $font-size-sm;
     margin-top: 0.25rem;
```

Comment:
I think we have a variable for this value

Reply:
changed !